wayback-webapp/src/main/webapp/WEB-INF/replay/Toolbar.jsp

```diff
@@ -320,6 +325,30 @@ function trackMouseMove(event,element) {
 
 </div>
 </div>
+   
+<script>
+    var x = sessionStorage.getItem("scrollX");
+    
+    (function($){
+        $("#yearChart").mCustomScrollbar({
+            axis: "x",
+            theme: "rounded-dots-dark",
+            setLeft: x,
```

Comment:
Yes, it should be removed. There is no need to set scroll-dragger here. It has been set inside callback function instead. I will update PR here and will fix it in BubbleCalendar too if needed.